WordPress/twentytwentyfive
# Add pattern: Hero Centered heading

**Description**

Fixes https://github.com/WordPress/twentytwentyfive/issues/90

- Adding pattern.
- Adding button styles in `theme.json`



**Screenshots**


https://github.com/user-attachments/assets/06d13415-3c25-418b-a7a5-313b4106619b




**Testing Instructions**

1. Create a page.
2. Add the pattern.
3. Confirm that it looks like the designs (spacing/colors/etc.).
